modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java

```diff
@@ -252,14 +289,40 @@ protected Reader generateSQL() {
 						createFileWriter(
 							new File(
 								BenchmarksPropsValues.OUTPUT_DIR,
-								"sample.sql")))) {
+								SampleSQLBuilderConstants.
```

Comment:
@slnn I remember that the "sample.sql" confused us in the past. Looking at the code here, it seems it's just the raw (not-yet-compressed) output. Is that correct?

Reply:
yes, it is, I merge create sql templates and the origin output of DataFactory in the sample.sql now

Reply:
@slnn Maybe we should change this file's name to something like "raw_output.sql"...

Reply:
OK